armi/resources/burn-chain.yaml

```diff
@@ -657,7 +659,7 @@ NP237:
     branch: 1.0
     halfLifeInSeconds: 67531821640000.0
     products:
-    - TH233
+    - PA233
```

Comment:
:+1:

Reply:
good news on this is that it doesn't affect anything except cases that have Np237 and a thorium decay chain because non-thorium cases just go straight to dump.